bulk/node_bulk_load-AssetRecord.csv

```diff
@@ -122,6 +122,7 @@ CGVEH-GLDRCP-00564,,Node,1,Coastal Profiling Glider 564,COASTAL GLIDER 200M,Tele
 CGVEH-GLDRCP-00583,,Node,1,Coastal Profiling Glider 583,COASTAL GLIDER 200M,Teledyne Webb,ASSY 302392-3,583,,,,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-00913,,Node,1,Coastal Glider 913,COASTAL GLIDER 1000M,Teledyne Webb,ASSY GLD-0089-D,913,,,,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-01145,,Node,1,Coastal Glider 1145,COASTAL GLIDER 1000M,Teledyne Webb,ASSY GLD-0089-D,1145,,,,,Woods Hole Oceanographic Institution
+CGVEH-GLDRGP-00380,,Node,1,Profiling Glider 380,,Teledyne Webb,ASSY 302392-3,380,,,,,Woods Hole Oceanographic Institution
```

Comment:
Looks okay.
There seems to be a mix of terminology with gliders.  Should this say "GLIDER GLOBAL PROFILING" after "Profiling Glider 380"?